src/Umbraco.Web.UI.Client/src/packages/ufm/components/content-name/content-name.element.ts

```diff
@@ -6,9 +6,8 @@ import { UmbId } from '@umbraco-cms/backoffice/id';
 import { UmbMediaItemRepository, UMB_MEDIA_ENTITY_TYPE } from '@umbraco-cms/backoffice/media';
 import { UmbMemberItemRepository, UMB_MEMBER_ENTITY_TYPE } from '@umbraco-cms/backoffice/member';
 
-const elementName = 'ufm-content-name';
-
-@customElement(elementName)
+// eslint-disable-next-line local-rules/enforce-umb-prefix-on-element-name
```

Comment:
I think we would do good by adding `"ufm"` as an allowed prefix in the eslint local rules.

Reply:
I've updated the eslint local rules.